dao/src/integ-test/java/uk/gov/moj/sdt/dao/TargetApplicationDaoTest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.moj.sdt.dao;
+
+import org.junit.Before;
+import org.junit.Test;
```

Comment:
Should the JUnit5 versions of these imports be used instead?  (I think the JUnit5 equivalent of Before is BeforeEach).

Reply:
Sorted